src/main/resources/zeebe-payments.json

```diff
@@ -0,0 +1,81 @@
+{
+  "index_patterns": [
+    "zeebe-payments_*"
+  ],
+  "order": 20,
+  "settings": {
+      "number_of_shards": "3",
+      "number_of_replicas": "0",
+      "queries": {
+        "cache": {
+          "enabled": "false"
+        }
+      }
+  },
+  "aliases": {
+    "zeebe-payments": {}
+  },
+  "mappings": {
```

Comment:
@ankita10r : Shouldn't we specify that the mapping is still dynamic (i.e. there is no need to statically define each variable)?